### ISSUE TITLE
Bug: No image for FF usage dashboard

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/posthog#16658]
## Bug description

<img width="574" alt="Screenshot 2023-07-19 at 11 53 15" src="https://github.com/PostHog/posthog/assets/84011561/960579fd-a731-4531-b473-e7430a3e8afc">

Would be nice to have a dashboard image here, as we do for the dashboard templates. 

Tried to add it myself, but looks like this is set up differently to the dashboard templates we made so I couldn't figure it out easily. It would have been a big effort for small impact, so: bug report. Sorry!

## How to reproduce

1.
2.
3.

## Environment

- [ ] PostHog Cloud US, project ID: [please provide from https://app.posthog.com/project/settings#project-id]
- [ ] PostHog Cloud EU, project ID: [please provide from https://eu.posthog.com/project/settings#project-id]
- [ ] PostHog Hobby self-hosted with `docker compose`, version/commit: [please provide]
- [ ] PostHog self-hosted with Kubernetes (deprecated, see ["Sunsetting Kubernetes support"](https://posthog.com/blog/sunsetting-helm-support-posthog)), version/commit: [please provide]

**Comment by Arun-chaitanya** [comment]
Hi @joethreepwood can you paste url once.

**Comment by joethreepwood** [comment]
Hi Arun - Not sure I understand your request here. Can you elaborate?

**Comment by neilkakkar** [comment]
Arun: https://app.posthog.com/feature_flags/ - you'll also have to enable the `ff-dashboard-templates` feature flag (create a new one with this name) to see this feature.

(Joe, they're asking so they know which page this is on)

**Comment by Arun-chaitanya** [comment]
Hi @neilkakkar @joethreepwood , not sure why but my view of the page looks like the following 
<img width="1440" alt="Screenshot 2023-07-24 at 5 08 41 PM" src="https://github.com/PostHog/posthog/assets/44457256/2d0ff1a9-13b6-410e-8833-743f140466e7">
Basically I am able to see only overview and history tab in my local. Do I need to take pull from any other branch.

**Comment by neilkakkar** [comment]
> you'll also have to enable the ff-dashboard-templates feature flag (create a new one with this name) to see this feature.

**Comment by Arun-chaitanya** [comment]
<img width="1440" alt="Screenshot 2023-07-24 at 11 37 15 PM" src="https://github.com/PostHog/posthog/assets/44457256/f19213d4-855e-4872-9fb8-7838a9cd7f1d">
Hi @neilkakkar I created a `ff-dashboard-templates` feature flag as suggested by you. But this is how it looks right now. I have all the tabs accept `Analysis` tab

**Comment by neilkakkar** [comment]
Ah, it's because you've created this in the demo project, not the 'main' project on your local dev, which controls which flags are toggled in all projects

**Comment by Arun-chaitanya** [comment]
@neilkakkar can you tell me how to run "main" project on local dev. I would love to contribute to more complex features also in future for posthog. At present many modules like "Experiment" are blocked for me in demo.

**Comment by neilkakkar** [comment]
Right, experiment is blocked on a license I'm afraid, but for the 'main' project, it should be the one you used to create the first org. You can edit this line: https://github.com/PostHog/posthog/blob/master/posthog/apps.py#L47 to be the demo project API key though if you want this to become the 'main' project. Just hardcode it.

**Comment by Arun-chaitanya** [comment]
Thanks
